src/Compilers/CSharp/Test/Semantic/Semantics/RefExtensionMethodsTests.cs

```diff
@@ -1965,6 +1965,26 @@ public static class Ext
             Assert.Equal(RefKind.In, symbol.RefKind);
         }
 
+        [Fact]
+        public void ParameterSymbolsRetrievedThroughSemanticModel_ModifierOrdering()
```

Comment:
>ParameterSymbolsRetrievedThroughSemanticModel_ModifierOrdering [](start = 20, length = 62)

This title is misleading. The symbols are not retrieved through the semantic model. I suggest following what `ParameterSymbolsRetrievedThroughSemanticModel_InExtensionMethods` do. I also suggest separating `this in` and `this ref`.

Reply:
That test is still useful, but I suggest moving it to the Symbols test project, and use `CompileAndVerify()` helper, making use of the `symbolValidator` parameter to check for the symbols.

---
In reply to: [157595248](https://github.com/dotnet/roslyn/pull/23643#discussion_r157595248) [](ancestors = 157595248)